lib/langAssembler.js

```diff
@@ -15,6 +15,14 @@ module.exports.assemble = assemble;
 function assemble(callback) {
     var localeDirectory = 'lang';
 
+    try {
+        Fs.statSync(localeDirectory);
+    }
+    catch (e) {
+        // No lang folder, callback immediately
+        return callback(null, 'no_lang_folder');
```

Comment:
I disagree with that, the return data, should be consistent, if the `results` variable is an object should always be an object. If the callback returns an error shouldn't indicate that the build has failed. You can set an error type or name and check for that name.

You could also check if the file exist before calling `assemble()`